google/cloud/storage/bucket.py

```diff
@@ -440,7 +440,7 @@ def delete_blobs(self, blobs, on_error=None, client=None):
                     raise
 
     def copy_blob(self, blob, destination_bucket, new_name=None,
-                  client=None):
+                  client=None, preserve_acl=False):
```

Comment:
The back-end *does* preserve the ACL during the `copyTo` operation:  the original poster in #2409 was asking for a way to disable that feature.  It would be more backward-compatible to make the default `True`.  In either case, if `preserve_acl` is `False`, we would then need to clear and save the target's ACL:

```python
    if not preserve_acl:
        new_blob.acl.reset()
        new_blob.acl.save()